hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -351,13 +351,19 @@ public static Server get() {
    * after the call returns.
    */
   private static final ThreadLocal<Call> CurCall = new ThreadLocal<Call>();
-  
+
+  private static final ThreadLocal<Long> CUR_CALL_STARTNANOS = new ThreadLocal<Long>();
+
```

Comment:
CUR_CALL_STARTNANOS Can it be deleted directly?

Reply:
Yes, Have fixed~ Thanks a lot.